src/castblock.cr

```diff
@@ -47,6 +48,10 @@ module Castblock
         @mute_ads = mute_ads.downcase == "true"
       end
 
+      if @skip_ads == false && (skip_ads = ENV["SKIP_ADS"]?)
+        @skip_ads = skip_ads.downcase == "true"
+      end
```

Comment:
I should really do something more standardized to handle parameters from env vars ^^